sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -717,6 +756,63 @@ export class KeyClient {
       return { value: result.value! };
     });
   }
+
+  /**
+   * Gets the rotation policy of a Key Vault Key.
+   *
+   * Example usage:
+   * ```ts
+   * let client = new KeyClient(vaultUrl, credentials);
+   * await client.setKeyRotationPolicy("MyKey", myPolicy);
+   * let result = await client.getKeyRotationPolicy("myKey");
+   * ```
+   *
+   * @param name - The name of the key.
+   * @param options - The optional parameters.
+   */
+  public getKeyRotationPolicy(
+    name: string,
+    options: GetRotationPolicyOptions = {}
+  ): Promise<KeyRotationPolicy | undefined> {
+    return withTrace("getRotationPolicy", options, async () => {
+      const policy = await this.client.getKeyRotationPolicy(this.vaultUrl, name);
+      if (policy.id) {
+        return keyRotationTransformations.generatedToPublic(policy);
+      }
+
+      return undefined;
+    });
+  }
+
+  /**
+   * Updates the rotation policy of a Key Vault Key.
+   *
+   * Example usage:
+   * ```ts
+   * let client = new KeyClient(vaultUrl, credentials);
+   * const setPolicy = await client.setKeyRotationPolicy("MyKey", myPolicy);
+   * ```
+   *
+   * @param name - The name of the key.
+   * @param policyParameters - The {@link KeyRotationPolicyParameters} for the policy.
+   * @param options - The optional parameters.
+   */
+  public setKeyRotationPolicy(
+    name: string,
+    policyParameters: KeyRotationPolicyParameters,
```

Comment:
These should be part of an "Options" bag, if not in `SetRotationPolicyOptions` for JS.